middleware/2fa.js

```diff
@@ -0,0 +1,287 @@
+const nearAPI = require('near-api-js');
+const crypto = require('crypto');
+const nacl = require('tweetnacl');
+const { creatorKeyJson } = require('./near');
+const { sendSms } = require('../utils/sms');
+const { sendMail } = require('../utils/email');
+const models = require('../models');
+const Sequelize = require('sequelize');
+const Op = Sequelize.Op;
+const password = require('secure-random-password');
+const SECURITY_CODE_DIGITS = 6;
```

Comment:
nit: let's have some space between imports and constants

Reply:
agree